zynqmp.mk

```diff
@@ -30,34 +38,61 @@ ifneq ($(PETALINUX_VER),2018.2)
 	$(error This makefile only support Petalinux 2018.2)
 endif
 
-petalinux-create: check-petalinux
-	@cd $(ROOT) && petalinux-create -t project -s $(BSP_PATH)
-	@cd $(PRJ_PATH) && petalinux-create -t apps --template install -n optee-client --enable
-	@cd $(PRJ_PATH) && petalinux-create -t apps --template install -n optee-test --enable
-	@#
+petalinux-create: check-petalinux	
+	@cd $(ROOT) && petalinux-create -n $(PLATFORM)-$(PETALINUX_VER) \
+	-t project -s $(BSP_PATH)
```

Comment:
suggest to add tab or space indentation for such multi-line command lines. Applies here and to many lines below:
```Patch
 	@cd $(ROOT) && petalinux-create -n $(PLATFORM)-$(PETALINUX_VER) \
-	-t project -s $(BSP_PATH)
+	    -t project -s $(BSP_PATH)
```